flutter/shell/platform/tizen/public/flutter_platform_view.h

```diff
@@ -30,6 +30,7 @@ class PlatformView {
 
   virtual void Dispose() = 0;
 
+  virtual void Offset(double left, double top) = 0;
```

Comment:
Could you provide a default implementation so that we don't break existing apps and plugins?